### ISSUE TITLE
Shortcut copy/paste like real keyboard!

### ISSUE THREAD

**Issue by DayatXD** [florisboard/florisboard#888]
add an feature to be able shorcut copy/paste by holding [?123] Button on the left bottom corner act like an [CTRL] then press [C] to copy, and [V] for paste.

Another option to shortcut this with gestures is by swipe from [SPACEBAR] + [C] to Copy Text!

Shortcut List:
CTRL + C = COPY
CTRL + V = PASTE
CTRL + X = CUT
CTRL + A = SELECT ALL
CTRL + Z = UNDO
CTRL + Y = REDO

**Comment by DayatXD** [comment]
![Screenshot_2021-05-16-14-09-42-357_com mayulive swiftkeyexi](https://user-images.githubusercontent.com/47524544/118387320-787c8f00-b650-11eb-97df-26be5c40b5dc.jpg)

Exi for SwiftKey an Xposed module for SwiftKey Keyboard have implemented this, and its work great!

**Comment by castrik** [comment]
yes, we can maybe use the bottom left key (which is used for switching to symbols) for this

like this, hold the ?123 key (at the bottom left) and type v to paste, etc.

it can work just like how the shift key works currently (hold shift and type words, let go of shift key) and the typed words will be uppercase, like an actual keyboard and keyboard switches back to lowercase after we let go

**Comment by ssssamstrup** [comment]
I think it'd be more useful to implement on the`Enter` key as symbols you could swipe between and choose. Chrooma keyboard does this for `Select all`, `Copy` and `Paste`. This could easily work with more and compliments the copy feature there right now.

For flexibility it would be cool to be able to edit the menu in settings, so you could choose yourself which to show and which to not.

I think implementing

**Comment by ssssamstrup** [comment]
nvm hadn't seen the undo/redo/copy/paste options when having text selected. Still think I'd be great to add Select all+Undo+Redo to the »Enter« longpress options.

Will create a seperate proposal for it.
